test/runnable/depsprot.sh

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env bash
 
-name=`basename $0 .sh`
+name="depsprot"
 dir=${RESULTS_DIR}/runnable
 dmddir=${RESULTS_DIR}${SEP}runnable
 deps_file="${dmddir}${SEP}${name}.deps"
 
+# custom error handling
+set +eo pipefail
```

Comment:
What’s the difference between `-` and `+`?

Reply:
This disables these settings as this test currently uses a custom approach to error handling. I might remove the custom approach here too, but I thought I leave that for another PR (and it's not really important if one file doesn't use pipefail. It's important that it's the default).